### ISSUE TITLE
private version

### ISSUE THREAD

**Issue by SilvaEngComp** [Lu-Yi-Hsun/iqoptionapi#187]
Hi! how can I get the private version? 
I'm analising your api to my boss and I need be right if that running right and we'll get the expected return. Do you have a trial option to teste the privete version?

**Comment by Lu-Yi-Hsun** [comment]
@SilvaEngComp 

If you buy now  300usd per month, and I can Customize api what you want.


You can test on public version, public maybe have some bug, the public version is maintain by anyone pull request who want do for free.

**Comment by Lu-Yi-Hsun** [comment]
@SilvaEngComp no worry any robot strategy you want. I can  customize any api you want.

**Comment by SilvaEngComp** [comment]
Van tou creste ano interface to windows?

**Comment by Lu-Yi-Hsun** [comment]
@SilvaEngComp 

The api can work in window.

**Comment by SilvaEngComp** [comment]
Yes,I know, but I'm taking about an implementation to final user or you Just provide the Api?

**Comment by Lu-Yi-Hsun** [comment]
@SilvaEngComp 

Yes 300 usd  only for api and some sample robot.

If you want build a great product for final user, it need other price.

**Comment by FabioAlbano** [comment]
Unfortunately, when the api go private, I was in the middle of my project, so I wasn't making any money yet. My question is, will the public version continue to be maintained, at least the basic buy/sell functions? My idea is to finish a basic app and sell it until I can afford to buy the private version to make a more complete app. Would it be safe or could the public version go down and stay non functional if IQ changes something?

**Comment by Lu-Yi-Hsun** [comment]
@FabioAlbano 

I am still thinking about what is the best way.....

Have been many people only using my free version.

If everyone only want the free version, iqoptionapi can not keep going better.

next step I will build the API for export option, that is free for who buy the iqoptionapi private version

**Comment by FabioAlbano** [comment]
$250 is more than R$1.000 reais, very expensive in Brazil. And $500.. No way for who is starting from zero clients. Maybe having one or two months period to pay it could be done.. But I understand that is not how it works. So I better abandon my project for now... But thanks anyway, I'm using personally for automated entries from MT4, so it's something

**Comment by JafferWilson** [comment]
@FabioAlbano I guess the buy/sell works for now.
You can try it out if you need any functionality. You can yourself write functions for it. 
I know about Brazil. But try to understand @Lu-Yi-Hsun  as well. He had been maintaining the API all on his own.
So he deserves something so that he can continue. He tried using the sponsored tag of Github, but there was no luck. I guess that might be the reason for him to go with the private version.

**Comment by FabioAlbano** [comment]
Yes, I completely agree. I even sent some money through the sponsor button, very modest values ​​because I was poor af... Maybe if everyone who used the API had contributed with what they could, things would be different.

> I guess the buy/sell works for now.

Hope so.. I will finish my application, I will charge very little because it is very basic, but I hope it helps me raise some dough.

**Comment by JafferWilson** [comment]
@FabioAlbano All the best for your application and I wish it work as you want it to. Plus, I am always there to help you if that is in my domain.... :P

**Comment by FabioAlbano** [comment]
@JafferWilson Thank you! Your domains are infinite levels above mine! I just started with Python.. cheers!

**Comment by Lu-Yi-Hsun** [comment]
Sorry I am still thinking what is best way, and how to balance...

**Comment by kkagill** [comment]
@Lu-Yi-Hsun 
As a suggestion, you can let people use your private version for free for 3~5 days that contains all of your features. 

Then what you can do is create a [pricing selection](https://www.teamviewer.com/en/buy-now/). Certainly, people like @FabioAlbano probably doesn't want to use everything in your private version. 

I bet most people just want to use the current api that can be maintained or ask for a new feature. 
If they liked your back trade system, then they can go for a higher pricing selection.
